.github/workflows/test.yml

```diff
@@ -17,9 +17,16 @@ jobs:
       - name: Check out repositrory code
         uses: actions/checkout@v2
 
-      - name: Docker commands
-        run: |
-          - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD $CI_REGISTRY
-          - docker build --tag $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA --tag $CI_REGISTRY_IMAGE:dev .
-          - docker push $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA
-          - docker push $CI_REGISTRY_IMAGE:dev
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+          
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKER_HUB_USERNAME }}/mumbleice:latest
```

Comment:
Can the tags key support multiple values? It would be great to push to a `latest` tag and a commit SHA hash tag. Also the image name should be `${{ env.CI_REGISTRY_IMAGE }}`, that way we configure it in settings and forks of this repo can still work